packages/x-components/static-docs/functional-doc/web-local-storage.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Interface X browser stored data and purposes
```

Comment:
@herrardo regarding the title, wouldn't it be better to be more explicit and reference **privacy** in some way since this will be the sidebar title?

```suggestion
title: Interface X data privacy and browser local storage
```